<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="whitefeather.xedge.testcases.PerformLoginAction" tests="13" time="6.043" ignored="0" failures="1" timestamp="11 Dec 2017 13:50:43 GMT" skipped="0" hostname="DESKTOP-TMQM6DS" errors="0">
  <testcase name="verifyUsernameInputField" time="0.068" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="clickLoginButton" time="3.261" classname="whitefeather.xedge.testcases.PerformLoginAction">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at whitefeather.xedge.testcases.PerformLoginAction.clickLoginButton(PerformLoginAction.java:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- clickLoginButton -->
  <testcase name="verifyLoginPageBackground" time="1.167" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="enterLoginId" time="0.376" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyClientAccountLogo" time="0.105" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyUsernameLabel" time="0.118" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyLoginButton" time="0.137" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyPasswordInputField" time="0.065" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="enterLoginPassword" time="0.140" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyPasswordLabel" time="0.114" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyPasswordWrapper" time="0.074" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyUsernamePlaceholder" time="0.206" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
  <testcase name="verifyPasswordPlaceholder" time="0.212" classname="whitefeather.xedge.testcases.PerformLoginAction"/>
</testsuite> <!-- whitefeather.xedge.testcases.PerformLoginAction -->
